ledger/participant-state/kvutils/src/main/protobuf/daml_kvutils.proto

```diff
@@ -71,6 +71,7 @@ message DamlSubmissionBatch {
   message CorrelatedSubmission {
     bytes submission = 1;
     string correlation_id = 2;
+    int64 estimated_interpretation_cost = 3;
```

Comment:
I'm surprised that you want to ship the estimated_interpretation_cost as part of a correlated submission, which is only used for batching. Given that we don't plan to use batching and pre-execution simultaneously, I wonder why we need this at all. Batching could just set the timing parameter to `None` for the underlying LedgerWriter.

Reply:
That's a good point. I have added this before making the timing parameter in `CommitMetadata` optional -- removing it.